fitbit-react/src/Pages/Authentication/Signup/SignUp.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Navbar from "../../../Components/Navbar/Navbar";
+import "../authentication.css";
+
+import { AiFillMail } from "react-icons/ai";
+import {
+  AiOutlineMail,
+  AiFillEyeInvisible,
+  AiFillEye,
+} from "../../../Assets/icons";
+import SignupForm from "./SignupForm";
+import AuthenticationBackground from "../AuthenticationBackground";
+function SignUp() {
+  return (
+    <div>
+      <Navbar />
+      <div className="spacer-3"></div>
+      <div className="wrapper">
+        <AuthenticationBackground />
+        <div className="authentication">
+          <div className="authentication-image-container">
+            <img src="images/signup.svg" alt="signup-image" />
```

Comment:
the alt tag doesnt need image to be mentioned in it as screen readers mention that its an image by default